tests/unit/firebaseRef.spec.js

```diff
@@ -2,9 +2,10 @@
 describe('firebaseRef', function () {
 
   var $firebaseRefProvider;
+  //TODO: Load this from env
```

Comment:
The unit tests run in Node.js. The e2e tests are the only thing that run in a browser environment I believe. So you should just be able to use `process.env` here.